src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -792,6 +792,25 @@ public int CompareTo(String strB)
             return false;
         }
 
+        [Pure]
+        public Boolean EndsWith(Char value)
+        {
+            return EndsWith(new string(value, 1), StringComparison.CurrentCulture);
```

Comment:
@jkotas, Isn't this the same approach of instantiating a new string while calling for the overloaded method in the class CompareInfo, line 756, (https://github.com/dotnet/coreclr/blob/b638af3a4dd52fa7b1ea1958164136c72096c25c/src/mscorlib/src/System/Globalization/CompareInfo.cs#L756)?